internal/apigen/cmd/api-generate-errors/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/redhat-developer/app-services-sdk-go/internal/apigen/metadata"
+)
+
+var repoMetadataPath string
+var repoErrorsPath string
+var accessToken string
+var clientPayloadStr string
+
+func init() {
+	flag.StringVar(&repoMetadataPath, "repo-metadata", "", "path to repo metadata JSON")
+	flag.StringVar(&repoErrorsPath, "errors-folder", "", "path to repo errors JSON files")
+}
+
+func main() {
+	flag.Parse()
+	if repoMetadataPath == "" {
+		log.Fatalln("Missing required flag: --repo-metadata")
+	}
+	if repoErrorsPath == "" {
+		log.Fatalln("Missing required flag: --errors-folder")
+	}
+
+	clientConfig, err := metadata.GetClientMetadata(repoMetadataPath)
+	if err != nil {
+		log.Fatalln("no metadata found:", err)
+	}
+
+	for id, metadata := range *clientConfig {
```

Comment:
I just made code compile but this is not functional - it is missing error structs per API and reading json files.